server/app/Models/User.php

```diff
@@ -448,7 +445,7 @@ public function hasRole($role)
      */
     private function beforeSave(array $options = array())
     {
-        $this->rules = $this->mergeRules();
+        static::$rules = $this->mergeRules();
```

Comment:
What's the difference between using `$this->` vs `static::` here? Just out of curiosity.